### ISSUE TITLE
A build error in VS 2015 (System.Collections.Immutable version mismatch)

### ISSUE THREAD

**Issue by ashmind** [cincuranet/ConfigureAwaitChecker#1]
```
1>CSC : error CS8032: An instance of analyzer ConfigureAwaitChecker.Analyzer.Conf
1>igureAwaitCheckerAnalyzer cannot be created from c:\Development\________\Develo
1>pment\______________\packages\ConfigureAwaitChecker.Analyzer.1.0.0-beta3\analyz
1>ers\dotnet\cs\ConfigureAwaitChecker.dll : Could not load file or assembly 'Syst
1>em.Collections.Immutable, Version=1.1.37.0, Culture=neutral, PublicKeyToken=b03
1>f5f7f11d50a3a' or one of its dependencies. The system cannot find the file spec
1>ified..
```

I think dotnet/roslyn#4831 might provide some info for this one.

**Comment by cincuranet** [comment]
Thanks. I'll look at it.

**Comment by danielmarbach** [comment]
I have the same issue.

**Comment by cincuranet** [comment]
Fixed in `1.0.0-beta4`. Pushing to NuGet.

**Comment by danielmarbach** [comment]
:+1:
